vroom/extensions.vroom

```diff
@@ -0,0 +1,75 @@
+Codefmt can be extended to support new formatters, using maktaba's extension
+registry.
```

Comment:
Could add some more details about what this means, why you would want to extend it vs. send a pull req to codefmt, etc., or link to those details. Don't want to block this bugfix on documentation details, though.